src/frontend/src/flows/recovery/recoveryWizard.json

```diff
@@ -0,0 +1,23 @@
+{
+  "en": {
+    "title": "Create your recovery phrase",
+    "paragraph": "This will take a few minutes. Make sure no one is watching your screen, and that you have a pen and paper or password manager ready. ",
+
+    "ok": "Create Recovery Phrase",
+    "skip": "Remind me later",
+
+    "what_is_phrase_q": "What is a recovery phrase?",
+    "what_is_phrase_a": "Your recovery phrase contains your Internet Identity number and a secret 24-word phrase",
+
+    "how_save_phrase_q": "How do I save my recovery phrase?",
+    "how_save_phrase_a_1": "Save in a password manager",
```

Comment:
nit: Imho this reads a bit funny due to the missing subject. I.e. "Save it in..."
But I'm not a native speaker, so no idea if this is actually an accurate impression.

Reply:
Update to "Save it in a password manager" as @frederikrothenberger suggested